packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -2975,6 +2983,97 @@ describe("IModelTransformerHub", () => {
     }
   });
 
+  for (const excludeBeforeTransformerInitialize of [true, false]) {
+    it(`should be able to handle entity recreation and a switching of exclude criteria ${excludeBeforeTransformerInitialize ? "before transformer initialize" : "after transformer initialize"}`, async () => {
```

Comment:
I will probably also setup a parent/child element recreation situation and make sure that a delete of the parent propagated to the target also deletes the child. Not sure if thats necessary or not since it feels like mostly testing deletes work as intended.